Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -37,17 +36,6 @@ public override void Initialize()
             base.Initialize();
 
             SubscribeNetworkEvent<DragDropRequestEvent>(HandleDragDropRequestEvent);
-
-            CommandBinds.Builder
-                .Bind(ContentKeyFunctions.TryPullObject,
-                    new PointerInputCmdHandler(HandleTryPullObject))
-                .Register<InteractionSystem>();
-        }
-
-        public override void Shutdown()
-        {
-            CommandBinds.Unregister<InteractionSystem>();
-            base.Shutdown();
```

Comment:
What did this removed block do, by the way?

Reply:
Systems need to unregister their type if they have every registered a bind. the old bind was registered under the type `InteractionSystem`, which made this line necessary. Since the bind is now registered under `SharedInteractionSystem`, this is no longer needed. (SharedInteractionSystem of course unregisters in its shutdown)